### PR TITLE
check style 

### PR DESCRIPTION
**Pull request by yhc777**
Fixes #1079 hippo4j-threadpool-infra-common

Changes proposed in this pull request:
- fix the hippo4j-threadpool-infra-common module code style 

> Check mailbox configuration when submitting. [Contributor Guide](https://hippo4j.cn/community/contributor-guide)

### CODE REVIEW

File: infra/common/src/main/java/cn/hippo4j/common/toolkit/IoUtil.java [hunk 1]
```diff
@@ -98,7 +100,7 @@ public static byte[] tryCompress(String str, String encoding) {
         try (GZIPOutputStream gzip = new GZIPOutputStream(out)) {
             gzip.write(str.getBytes(encoding));
         } catch (Exception e) {
-            e.printStackTrace();
+            log.info("gzip write is fail:{}", JSONUtil.isJson(e.toString()));
```
Comment (magestacks) [inline infra/common/src/main/java/cn/hippo4j/common/toolkit/IoUtil.java]: Why do you need json processing of exception stack?

File: infra/common/src/main/java/cn/hippo4j/common/toolkit/IoUtil.java [hunk 2]
```diff
@@ -47,6 +48,7 @@
  *
  * @author nacos
  */
+@Slf4j(topic = "IoUtil")
```
Comment (magestacks) [inline infra/common/src/main/java/cn/hippo4j/common/toolkit/IoUtil.java]: Subsequent unified processing, this PR will not be added

File: infra/common/src/main/java/cn/hippo4j/common/toolkit/IoUtil.java (hunk 1 above)
Comment (yhc777) [reply infra/common/src/main/java/cn/hippo4j/common/toolkit/IoUtil.java]: write casually. ok， i will update and commit!

Comment (magestacks) [reply infra/common/src/main/java/cn/hippo4j/common/toolkit/IoUtil.java]: should print error level logs

Comment (yhc777) [reply infra/common/src/main/java/cn/hippo4j/common/toolkit/IoUtil.java]: ok

**Review by magestacks — APPROVED** [review]

**Pull request merged**